packages/core/admin/admin/src/features/Auth.tsx

```diff
@@ -122,27 +121,6 @@ const AuthProvider = ({
     navigate('/auth/login');
   }, [dispatch, navigate]);
 
-  /**
-   * Fetch data from storages on mount and store it in our state.
-   * It's not normally stored in session storage unless the user
-   * does click "remember me" when they login. We also need to renew the token.
-   */
-  React.useEffect(() => {
```

Comment:
Token renewal has been moved to packages/core/admin/admin/src/utils/baseQuery.ts